withings_sync/garmin.py

```diff
@@ -1,56 +1,52 @@
 """This module handles the Garmin connectivity."""
+
+import io
 import logging
-import garth
 import os
-import io
 
-log = logging.getLogger("garmin")
+import garth
 
+log = logging.getLogger("garmin")
 
-HOME = os.getenv('HOME', '.')
-GARMIN_SESSION = os.getenv('GARMIN_SESSION', HOME + '/.garmin_session')
+HOME = os.getenv("HOME", ".")
+GARMIN_SESSION = os.path.join(HOME, ".garmin_session")
```

Comment:
```suggestion
GARMIN_SESSION = os.getenv('GARMIN_SESSION', os.path.join(HOME, ".garmin_session"))
```

don't break the GARMIN_SESSION override.